packages/client/src/components/Actions/components/ActionRequired.js

```diff
@@ -20,13 +20,17 @@ const ActionRequired = ({
         return <TokenView actionView={actionView} safeData={safeData} />;
       case ACTION_TYPES.TRANSFER_NFT:
         return <NFTView actionView={actionView} safeData={safeData} />;
-      case ACTION_TYPES.ADD_OWNER:
+      case ACTION_TYPES.ADD_SIGNER_UPDATE_THRESHOLD:
```

Comment:
is it still possible to trigger just `ADD_OWNER` ? if so maybe this should be 

```
case ACTION_TYPES.ADD_SIGNER_UPDATE_THRESHOLD:
case ACTION_TYPES.ADD_OWNER:
```

Reply:
From the UI it's not possible, but I see that it still exists in the contracts.
@jacksonConrad do we intend to remove AddSigner and RemoveSigner actions from the TreasuryActions.cdc?